api/v1alpha5/metal3datatemplate_types.go

```diff
@@ -202,6 +202,11 @@ type NetworkDataLinkBond struct {
 	// balance-rr, active-backup, balance-xor, broadcast, balance-tlb, balance-alb, 802.3ad
 	BondMode string `json:"bondMode"`
 
+	// +kubebuilder:validation:Enum="layer2";"layer3+4";"layer2+3"
+	// Selects the transmit hash policy used for port selection in balance-xor and 802.3ad modes
+	// +optional
+	BondXmitHashPolicy string `json:"bondXmitHashPolicy"`
```

Comment:
We shouldn't add the new field to v1alpha5 API IMO but only to v1beta1, the former should stay the same, only conversion should be changed.

/cc @kashifest opinions?

Reply:
@furkatgofurov7 
remove the bondXmitHashPolicy in v1alpha5 ?

Reply:
@shibaPuppy can you update your PR then which adds the new field in v1b1 api only? v1a5 is deprecated and will be removed gradually #971